misaligned.c

```diff
@@ -16,6 +18,7 @@ int printColorMap() {
 int main() {
     int result = printColorMap();
     assert(result == 25);

```

Comment:
The tests should include how color/pair number are printed including their alignment also. Please consider the pair with single and double digits and its effect on alignment.